docs/design_docs/011-historacle-pricing.md

```diff
@@ -53,18 +54,18 @@ We define two Maximum values, which correspond to the most we will store of a me
 
 ### Historic Data Table
 
-| Name           | Description                                                          | How often it is recorded      | When it is pruned                     |
-| -------------- | -------------------------------------------------------------------- | ----------------------------- | ------------------------------------  |
-| `Price Stamp`  | A price for an asset recorded at a given block                       | Every `Historic Stamp Period` | After `Maximum Price Stamps` is met.  |
-| `Median Stamp` | Of a given asset & block, the median of the stored `Price Stamps`    | Every `Median Stamp Period`   | After `Maximum Median Stamps` is met. |
+| Name           | Description                                                       | How often it is recorded      | When it is pruned                     |
+| -------------- | ----------------------------------------------------------------- | ----------------------------- | ------------------------------------- |
+| `Price Stamp`  | A price for an asset recorded at a given block                    | Every `Historic Stamp Period` | After `Maximum Price Stamps` is met.  |
+| `Median Stamp` | Of a given asset & block, the median of the stored `Price Stamps` | Every `Median Stamp Period`   | After `Maximum Median Stamps` is met. |
 
 ### Proposed API
 
 Modules will have access to the following `keeper` functions from the `x/oracle` module:
 
-- `HistoricMedian(denom) (sdk.Dec, error)` returns the median price of an asset in the last `Pruning Period`
-- `WithinHistoricDeviation(denom) (bool, error)` returns whether or not the current price of an asset is within the `Standard Deviation around the Median`.
-- `MedianOfMedians(denom string, blockNum int) sdk.Dec` returns the Median of the all the Medians recorded within the past `blockNum`.
+- `HistoricMedian(denom) ([]sdk.Dec, error)` returns list median prices of an asset in the last `Pruning Period`
+- `WithinHistoricDeviation(denom) (bool, error)` returns whether or not the current price of an asset is within the Standard Deviation around the Median.
+- `MedianOfMedians(denom string, blockNum int) sdk.Dec` returns the Median of the all the Medians recorded within the past `blockNum`. TODO: what does it mean "within the past blockNum"?
```

Comment:
For example `MedianOfMedians(uumee,2500)` makes a computation involving all medians stamped "in the last 2500 blocks"

Reply:
@toteki we were actually thinking of using a range here as well, something like
`MedianOfMedians(denom string, fromBlock, toBlock int)`
cc: @rbajollari

Reply:
Please see: https://github.com/umee-network/umee/blob/a90346e3b8b721e8a4dc78707d5b3cd2db91d20b/x/oracle/keeper/historic_price.go#L192 

In https://github.com/umee-network/umee/blob/a90346e3b8b721e8a4dc78707d5b3cd2db91d20b/x/oracle/keeper/historic_price.go#L192

Might want to update that here

Reply:
Shouldn't we use `MedianOfMedians(denom, number_of_stamps)` instead?

Reply:
So I don't think leverage will be using min/max blocks. Might be useful on ojo or similar though

Reply:
good, so leaving as proposed here: `MedianOfMedians(denom, number_of_stamps)`